backends/advanced/src/advanced_omi_backend/services/memory/providers/chronicle.py

```
@@ -476,6 +476,202 @@ def shutdown(self) -> None:
+                and isinstance(action.get("text"), str)
+            ]
+
+            text_to_embedding = {}
+            if texts_needing_embeddings:
+                try:
+                    embeddings = await asyncio.wait_for(
+                        self.llm_provider.generate_embeddings(texts_needing_embeddings),
+                        timeout=self.config.timeout_seconds,
+                    )
+                    text_to_embedding = dict(
+                        zip(texts_needing_embeddings, embeddings)
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)